hassio/docker/addon.py

```diff
@@ -88,6 +88,12 @@ def environment(self):
                 'ALSA_INPUT': self.addon.audio_input,
             })
 
+        # Provide options for legacy add-ons
+        if self.addon.legacy:
+            addon_env.update({
+                **self.addon.options
+            })
+
```

Comment:
ENV support only one depth. We should make sure, that there is no nested struct inside. `**{value: key in value, key for options.items() if isinstance(key, str, int)}`